packages/nextjs/src/server/clerkMiddleware.ts

```diff
@@ -1,3 +1,5 @@
+import { AsyncLocalStorage } from 'node:async_hooks';
```

Comment:
By using `node:async_hooks`, it uses the `AsyncLocalStorage` polyfill from Next.js. TBH it's a controversy from what Vercel docs mention, in which `async_hooks` would also be polyfilled: https://vercel.com/docs/functions/runtimes/edge-runtime#compatible-node.js-modules

![CleanShot 2024-07-04 at 10 26 57](https://github.com/clerk/javascript/assets/48022589/9b14a607-8ad0-45c5-8018-f9e408fd3070)